### PR TITLE
Fix request body type

### PR DESCRIPTION
**Pull request by philippguertler**
When the raml stated a request body with `application/json` but no schema was given, the emitted TS interface now states `body: any;` instead of `body?: undefined`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,8 +30,8 @@
     "build": "gulp build",
     "build-tsc": "tsc -p tsconfig.es5.json && tsc -p tsconfig.es6.json",
     "prepublish": "gulp default",
-    "test": "mocha --require ts-node/register --require source-map-support/register src/**/*.spec.ts",
-    "watch-tests": "mocha --watch --reporter min --require ts-node/register --require source-map-support/register src/**/*.spec.ts",
+    "test": "mocha --require ts-node/register --require source-map-support/register src/**.spec.ts src/**/*.spec.ts",
```
Comment (leonadler) [inline package.json]: you can remove the first pattern.
"src/**/*.spec.ts" already means "files ending in .spec.ts in src, with or without subdirectories"

File: src/renderers/typescript-renderer.spec.ts [hunk 2]
```diff
@@ -620,6 +626,62 @@ describe('TypescriptModelRenderer', () => {
...
+                        };
+                    };
+                    PATCH: { };
+                    PUT: { };
+                    DELETE: { };
+                }
+
+            `);
+        })
+
```
Comment (leonadler) [inline src/renderers/typescript-renderer.spec.ts]: Very nice addition! Glad to see my code is sufficiently manageable :-)

:+1:

File: package.json (hunk 1 above)
Comment (philippguertler) [reply package.json]: Yes I thought so too, but it only matched subdirectories of `src`, but not files in the `src` folder itself.
I googled it now, and I had the same problem as this guy: https://github.com/mochajs/mocha/issues/1828

Putting the argument in single quotes works now.

File: src/renderers/typescript-renderer.spec.ts (hunk 2 above)
Comment (philippguertler) [reply src/renderers/typescript-renderer.spec.ts]: Thanks, it's a very useful library, that's why I try to rely on it as much as possible.

**Pull request merged**